serving/docs/lmi/tutorials/tnx_aot_tutorial.md

```diff
@@ -0,0 +1,157 @@
+
+### Step 2: Pull the docker image
+
+Refer [here](https://github.com/aws/deep-learning-containers/blob/master/available_images.md#large-model-inference-containers) for the latest LMI NeuronX DLC and pull the image.
+
+For example:
+
+```bash
+# Login to ECR
+aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-west-2.amazonaws.com
+
+# Download docker image
+docker pull 763104351884.dkr.ecr.us-west-2.amazonaws.com/djl-inference:0.26.0-neuronx-sdk2.16.0
+
+```
+
+### Step 3: Set the environment variables:
+
+These below configurations helps you configure the inference optimizations parameters. You can check all the configurations of LMI NeuronX handler [in our docs](https://docs.djl.ai/docs/serving/serving/docs/lmi/configurations_large_model_inference_containers.html#transformers-neuronx-doc).
+You will save these as a `serving.properties` text file that will define the models compilation configuration.
+
+___serving.properties___
+```
+engine=Python
+option.model_id={{huggingface_model_id/s3url/directory}}
+option.entryPoint=djl_python.transformers_neuronx
+option.tensor_parallel_degree=24
+option.n_positions=512
+option.rolling_batch=auto
+option.max_rolling_batch_size=8
+option.enable_mixed_precision_accumulation=true
+option.model_loading_timeout=12000
+option.save_mp_checkpoint_path=/opt/ml/input/data/training/partition-test
+```
+
+### Step 4: Setup directories and model artifacts for compilation
+
+In this tutorial using Deep Learning AMI Neuron PyTorch 1.13 (Ubuntu 20.04) on an inf2.48xlarge and assume we are working the default home directory `~/`. If you are using a HuggingFace `model_id` or s3 url for your model then you can move on to the next step.
+
+For downloading a model to a local directory we will be using the following script (model and directory can be updated for your use case):
+
+___download.py___
+```python
+from huggingface_hub import snapshot_download
+from pathlib import Path
+
+# - This will download the model into the ./model directory where ever the script is running
+local_model_path = Path("./model")
+local_model_path.mkdir(exist_ok=True)
+model_name = "meta-llama/Llama-2-70b-hf"
+# Only download safetensors checkpoint files
+allow_patterns = ["*.json", "*.safetensors", "*.pt", "*.txt", "*.model", "*.tiktoken"]
+
+# - Leverage the snapshot library to download the model since the model is stored in repository using LFS
+snapshot_download(
+    repo_id=model_name,
+    local_dir=local_model_path,
+    local_dir_use_symlinks=False,
+    allow_patterns=allow_patterns,
+    token="hf_YOUR_TOKEN_VALUE",
+)
+```
+
+We will run the `download.py` script defined above in order to download a local copy of the model into the `~/model` directory.
+
+```bash
+pip install huggingface_hub pathlib
+python3 download.py
+```
+
+After the download completes the `~/model` directory will contain the model artifacts that we can use for compilation.
```

Comment:
in the below command, you didn't use this directory for mounting

Reply:
That is correct - the model folder attaches in this line.
  -v /home/ubuntu/:/opt/ml/input/data/training \  # Line 119
The model directory when this is followed verbatim ends up at /home/ubuntu/model, with the serving.properties in /home/ubuntu. The one thing I will have to double check is that you can just put model in the option.model_id.